roles/aws/delete-instance/tasks/dns.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Same as before re: existing dns playbooks

Reply:
will do in future